### ISSUE TITLE
Missing Community Standards

### ISSUE THREAD

**Issue by Bashamega** [ChrisTitusTech/winutil#2504]
Hello:)

I have noticed that this repository is missing some important files to help more people contribute.
These are the missing files:

1. Code of Conduct, which can help reduce the number of prs that are not following the guidelines.
2. Contributing guide which can help contributors set up the repo locally to contribute.
3. Security policy which can help people report security vulnerabilities.

**Comment by Real-MullaC** [comment]
> 2\. Contributing guide which can help contributors set up the repo locally to contribute.

This can be found here - https://christitustech.github.io/winutil/contribute/#walk-through

**Comment by Bashamega** [comment]
> > 2\. Contributing guide which can help contributors set up the repo locally to contribute.
> 
> This can be found here - https://christitustech.github.io/winutil/contribute/#walk-through
> 

Correct, but it is more appropriate to ad a markdown file for it inside of the root directory

**Comment by Real-MullaC** [comment]
> > > 2\. Contributing guide which can help contributors set up the repo locally to contribute.
> > 
> > This can be found here - https://christitustech.github.io/winutil/contribute/#walk-through
> > 
> 
> Correct, but it is more appropriate to ad a markdown file for it inside of the root directory 

We put it on the docs that is one of the reasons we added the docs.

**Comment by MyDrift-user** [comment]
Hey @Bashamega 
I understand what you mean and I agree that this would be a great change if we can implement it correctly with the existing docs.

@Real-MullaC The way he suggests it we would follow the standards of github. You can see it [here](https://github.com/ChrisTitusTech/winutil/community).

That way we could even have the nav entry on the github repo page, like following:
![image](https://github.com/user-attachments/assets/dc86bfdc-4ba7-49cf-93c4-699e675a39fc)

**Comment by NotYourAverageGamer** [comment]
2. Perhaps we could use GitHub Actions to copy `/docs/CONTRIBUTING.md` to the base directory of the repo, whenever changes are made to `CONTRIBUTING.md`, so that the two files stay synced?


<details>
<summary>Rough GPT example (actions noob here):</summary>

```
name: Sync CONTRIBUTING.md

on:
  push:
    paths:
      - 'docs/CONTRIBUTING.md'
  workflow_dispatch:

jobs:
  sync-contributing:
    runs-on: ubuntu-latest

    steps:
    - name: Checkout repository content
      uses: actions/checkout@v3

    - name: Copy CONTRIBUTING.md to root directory
      run: cp docs/CONTRIBUTING.md CONTRIBUTING.md

    - name: Commit and push changes
      run: |
        git config --global user.name "github-actions[bot]"
        git config --global user.email "github-actions[bot]@users.noreply.github.com"
        git add CONTRIBUTING.md
        git commit -m "Sync CONTRIBUTING.md from /docs to root"
        git push
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

</details>

**Comment by Bashamega** [comment]
Great idea @NotYourAverageGamer

**Comment by Bashamega** [comment]
should i work on the github actions?

**Comment by MyDrift-user** [comment]
@Bashamega No need for that, I will fix it in #2481

you can reference the file instead of cloning it using github action which would be the more efficient way
